master/reference/cni-plugin/configuration.md

```diff
@@ -418,6 +418,23 @@ Check out the usage guide on [assigning IP addresses based on
 topology]({{site.baseurl}}/{{page.version}}/networking/assigning-ip-addresses-topology)
 for a full example.
 
+### CNI network configuration lists
+
+CNI 3.3.0 [spec](https://github.com/containernetworking/cni/blob/spec-v0.3.0/SPEC.md#network-configuration-lists) supports "chaining" multiple cni plugins together and {{site.prodname}} supports this as well. {{site.prodname}} enables the portmap plugin by default which is required to implement Kubernetes portmap functionality. This can be disabled by removing the portmap section from the CNI network configuration in the Calico manifests.
+
+ ```json
+        {
+          "type": "portmap",
+          "snat": true,
+          "capabilities": {"portMappings": true}
+        }
+```
```

Comment:
I think we should tag this with `{:.no-select-button}` so that users don't get the "copy" button in the top right corner.

e.g.: https://github.com/projectcalico/calico/search?q=.no-select-button&unscoped_q=.no-select-button

Reply:
done